packages/block-library/src/index.native.js

```diff
@@ -96,9 +96,9 @@ export const coreBlocks = [
 	textColumns,
 	verse,
 	video,
-].reduce( ( memo, block ) => {
-	memo[ block.name ] = block;
-	return memo;
+].reduce( ( memoAccumulator, block ) => {
```

Comment:
A similar note here. It feels like `memoAccumulator` could be shortened to `accumulator` and it will read well enough.